docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -0,0 +1,104 @@
+[role="xpack"]
+[[dfa-regression]]
+== {regression-cap}
+
+{reganalysis-cap} is a {ml} process for estimating the relationships among 
+a number of {feature-vars} and a {depvar}, then making further predictions based 
+on the described relationship.
+
+For example, suppose we are interested in finding the relationship between 
+apartment size and monthly rent in a city. In this example, the rent is the 
+{depvar} while the apartment size is the {feature-var} and we try to find out 
+the relationship between them in order to make further predictions. Our 
+imaginary dataset consists of three data points:
+
+|===
+| Size (m2) | Monthly rent 
+| 44        | 1600
+| 24        | 1055
+| 63        | 2300
+|===
+
+After the model described the relationship between the {feature-vars} and the 
+{depvar}, it can predict the monthly rent of a hundred square meter-size 
+apartment.
+
+This is a simple example, usually {regression} problems are multi-dimensional, 
+so the relationships that {reganalysis} tries to find are between multiple 
+{feature-vars} and the {depvar}. To extend our example, a more complex 
+{reganalysis} could take into account further {feature-vars} like the location 
+of the apartment in the city, on which floor it is, and whether the apartment 
+has a riverside view or not and so on.
+
+
+[discrete]
+[[dfa-regression-features]]
+=== {feature-vars-cap}
+
+{feature-vars-cap} are the values that the {depvar} value depends on. If one or 
+more of the {feature-vars} changes, the {depvar} value also changes. There are 
+three different types of {feature-vars} that you can ship to the {regression} 
+algorithm:
+* Numerical. In our example, the size of the apartment was a 
+  numerical {feature-var}.
+* Categorical. A variable that can have one value from a set of values. The 
+  value set has a fixed and limited number of possible items. In the example, 
+  the location of the apartment in the city (borough) is a categorical variable.
+* Boolean. The riverside view in the example is a boolean value because an 
+  apartment either has a riverside view or doesn't have one.
+Arrays are not supported.
+
+
+[discrete]
+[[dfa-regression-supervised]]
+=== Training the {regression} algorithm
+
+{regression-cap} is a supervised machine learning process, which means that you 
+need to supply a labeled training dataset that has some {feature-vars} and a 
```

Comment:
In my opinion, at some point we'll need more information about how they generate a labelled training dataset, since so far I haven't seen that covered.

Reply:
I agree. I'll ask around and come up with a solution.